csharp-algorithms/FindAllPositionsOfAString/FindAllPositionsOfAString/Samples/SearchingSamples.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace FindAllPositionsOfAString.Samples;
+
+public record SearchPair(string Text, string SearchText);
+
+public static class SearchingSamples
+{
+    private static string JuliusCaesar = JuliusCaesarText.Read();
+
+    private readonly static SearchPair[] searchPairs =
```

Comment:
Incorrect modifier declaration order - should be `private static readonly`

Reply:
Corrected